autoptsclient_common.py

```diff
@@ -549,10 +561,16 @@ def wrapper(*args):
         end_time = str(round(datetime.timedelta(
             seconds=end_time).total_seconds(), 3))
 
-        print("{}".format(test_case.status).ljust(16) +
-              end_time.rjust(len(end_time)) +
-              retries_msg.rjust(len("#{}".format(retries_max)) + margin) +
-              regression_msg.rjust(len("REGRESSION") + margin))
+        result = ("{}".format(test_case.status).ljust(16) +
+                end_time.rjust(len(end_time)) +
+                retries_msg.rjust(len("#{}".format(retries_max)) + margin) +
+                regression_msg.rjust(len("REGRESSION") + margin))
+
+        if sys.stdout.isatty():
+            output_color = get_the_color(test_case.status)
```

Comment:
I actually ran the PR and found the issue: it should be get_result_color().

Please update and submit again.

Reply:
Done, sorry for a silly mistake.